sys/Makefile.dep

```diff
@@ -498,6 +498,10 @@ ifneq (,$(filter devfs,$(USEMODULE)))
   USEMODULE += vfs
 endif
 
+ifneq (,$(filter vfs_default,$(USEMODULE)))
+  USEMODULE += vfs
```

Comment:
vfs_auto_mount gets added as a default module if vfs is selected.
We could change it to make vfs_default select it instead

Reply:
No strong preference, but it might make sense. Having XFA components active only when there is an active user generally saves a few bytes, and having default-modules pulled in through a module that obviously pulls in things vs. modules that just provide some API and code doesn't sound wrong either..

Reply:
I think it makes sense to use `vfs_default` for selecting the auto-mounter.